src/shared/helpers/getActualDataList.ts

```diff
@@ -17,10 +18,13 @@ export const getActualDataList = ({
   actualDelayInDays,
```

Comment:
Can we replace 'getActualDataList' with a more telling name?

Reply:
Seems like it's good enough. It can sort both events and courses and filter them to exclude stale. So naming is pretty universal. If you disagree please suggest something.

Reply:
My bad, it can't sort courses, cuz they have startDate instead of date. Anyway i think refactoring this func is beyond this task

Reply:
I don't get this either, can you add JSDoc comments which explains what this actually doing?

![image](https://github.com/rolling-scopes/site/assets/114234698/a386ead7-6d9d-4c07-8b6d-0f65fc9b6ba8)

Reply:
Will make it in #400